src/Swashbuckle.AspNetCore.Annotations/AnnotationsParameterFilter.cs

```diff
@@ -31,8 +31,7 @@ private void ApplyPropertyAnnotations(OpenApiParameter parameter, PropertyInfo p
         private void ApplyParamAnnotations(OpenApiParameter parameter, ParameterInfo parameterInfo)
         {
 
-            var swaggerParameterAttribute = parameterInfo.GetCustomAttributes<SwaggerParameterAttribute>()
-                .FirstOrDefault();
+            var swaggerParameterAttribute = parameterInfo.GetCustomAttribute<SwaggerParameterAttribute>();
```

Comment:
What is the stack trace of the exception that occurs without this change? Odd that one works but "first of possibly many" doesn't.

Reply:
It's a thing that is only happening for MinimalApi when the property has a SwaggerParameterAttribute on it. I think the problem is right here: 
![image](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/assets/6615999/4fd913db-3bb5-46c5-830f-4a7c94a86b23)

It is not able to cast the array of Attributes to a Ienumerable of SwaggerParameterAttribute. Since the SwaggerParamterAttribute can only be applied once for each parameter then there is no problem in adding it

The stackTrace is this one:
Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[1]
      An unhandled exception has occurred while executing the request.
      Swashbuckle.AspNetCore.SwaggerGen.SwaggerGeneratorException: Failed to generate Operation for action - HTTP: POST /fruit/{id}. See inner exception
       ---> System.InvalidCastException: Unable to cast object of type 'System.Attribute[]' to type 'System.Collections.Generic.IEnumerable`1[Swashbuckle.AspNetCore.Annotations.SwaggerParameterAttribute]'.
         at System.Reflection.CustomAttributeExtensions.GetCustomAttributes[T](ParameterInfo element)
         at Swashbuckle.AspNetCore.Annotations.AnnotationsParameterFilter.ApplyParamAnnotations(OpenApiParameter parameter, ParameterInfo parameterInfo) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.Annotations\AnnotationsParameterFilter.cs:line 34
         at Swashbuckle.AspNetCore.Annotations.AnnotationsParameterFilter.Apply(OpenApiParameter parameter, ParameterFilterContext context) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.Annotations\AnnotationsParameterFilter.cs:line 18
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateParameterAsync(ApiParameterDescription apiParameter, SchemaRepository schemaRepository) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 599
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateParametersAsync(ApiDescription apiDescription, SchemaRepository schemaRespository, Func`3 parameterGenerator) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 490
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateParametersAsync(ApiDescription apiDescription, SchemaRepository schemaRespository) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 508
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateOperationAsync(ApiDescription apiDescription, SchemaRepository schemaRepository, Func`3 parametersGenerator, Func`3 bodyGenerator, Func`3 applyFilters) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 322
         --- End of inner exception stack trace ---
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateOperationAsync(ApiDescription apiDescription, SchemaRepository schemaRepository, Func`3 parametersGenerator, Func`3 bodyGenerator, Func`3 applyFilters) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 345
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateOperationAsync(ApiDescription apiDescription, SchemaRepository schemaRepository) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 373
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GenerateOperationsAsync(IEnumerable`1 apiDescriptions, SchemaRepository schemaRepository) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 268
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GeneratePathsAsync(IEnumerable`1 apiDescriptions, SchemaRepository schemaRepository, Func`3 operationsGenerator) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 207
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GeneratePathsAsync(IEnumerable`1 apiDescriptions, SchemaRepository schemaRepository) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 229
         at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwaggerAsync(String documentName, String host, String basePath) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.SwaggerGen\SwaggerGenerator\SwaggerGenerator.cs:line 56
         at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider) in C:\Git\Github\Swashbuckle.AspNetCore\src\Swashbuckle.AspNetCore.Swagger\SwaggerMiddleware.cs:line 64
         at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)

Reply:
It's strange indeed!